### PR TITLE
fix tsconfig.json in web template

### PR DESCRIPTION
**Pull request by PeterChen1997**
The include part should includes the ./types folder instead of ../types

### CODE REVIEW

**Comment by Tobbe** [comment]
@PeterChen1997 Thanks for this PR.
This looks like an obvious fix, but I'm surprised no one has reported this earlier. What's the easiest way to verify this fix? I.e. how can I see that something's broken with the current config, and then see it working with your change?

**Comment by PeterChen1997** [comment]
Oh, that's easy to test! I mentioned it in this issue https://github.com/redwoodjs/redwood/issues/5415

I add this in types/typings.d.ts but get this this error

```ts
declare module '*.module.scss'
```
image


after I fix the tsconfig, it works fine!

File: packages/create-redwood-app/template/web/tsconfig.json [hunk 1]
```diff
@@ -31,6 +31,6 @@
     "src",
     "../.redwood/types/includes/all-*",
     "../.redwood/types/includes/web-*",
-    "../types"
+    "./types"
```
Comment (Tobbe) [inline packages/create-redwood-app/template/web/tsconfig.json:34]:
```suggestion
    "../types",
    "./types"
```

If you look at the `types/*` entry under `paths` above you'll notice that it lists both `./types` and `../types`. So I think we should have both here as well. What do you think?

**Comment by Tobbe** [comment]
> Oh, that's easy to test!

Thanks! I added a review comment to the code

File: packages/create-redwood-app/template/web/tsconfig.json (hunk 1 above)
Comment (PeterChen1997) [reply packages/create-redwood-app/template/web/tsconfig.json:35]: That's cool, done!

**Pull request merged**